test/inspector/test-async-hook-setup-at-signal.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+common.skipIf32Bits();
+const helper = require('./inspector-helper');
+const assert = require('assert');
+
+const script = `
+process._rawDebug('Waiting until a signal enables the inspector...');
+const waiting = setInterval(setupIntervalIfDebugged, 50);
+
+function setupIntervalIfDebugged() {
+ if (!process.binding('inspector').isEnabled()) return;
+ clearInterval(waiting);
+ // Introduce a small delay to allow V8 engine to correctly setup
+ // V8 Inspector async-task infrastructure.
+ setTimeout(setupIntervalWithBreak, 100);
```

Comment:
I think I found a viable solution - introducing another `debugger;` break to sync between the debugged process, V8 inspector and the test session. See 7d80c1277.